### ISSUE TITLE
Zooeybot stuck at rerolling summon

### ISSUE THREAD

**Issue by Einhalkzt** [Masuzu/ZooeyBot#698]
the boll will get stuck when it's trying to retreat from the trial fight.
what happen:
- Go into trial
- Go to quest page
- Click retreat
- Click another confirmation
- "You have retreat from the battle" pops, and the bot just stuck there doing nothing not even refreshing just stuck, not clicking the ok too.

**Comment by Shamool** [comment]
Are you on the chromium supported version? I got that error just now too on ver 39. Just switched to 37, will tell you how it goes

**Comment by Einhalkzt** [comment]
im using chromium v67.0.3369.0
i switch back to using zooeybot v 2.11.3 and it seems to be working fine again
2.11.6 broke the summon reroll for me

i'll try running it for a long time and see how it goes

**Comment by Shamool** [comment]
RIP, I don't have V2.11.3. I don't know how to get it either

**Comment by Masuzu** [comment]
Hello,
If after navigating to the quest menu from the trial battle, if you do not have any popup such as 
![image](https://user-images.githubusercontent.com/5464705/37866177-368272b6-2f87-11e8-8a32-8ab717e2bcb5.png)
I guess there should be another issue on your side?

**Comment by Shamool** [comment]
It appeared, I just pressed okay on the popup while trying to snip by accident

**Comment by Einhalkzt** [comment]
yeah im experiencing the same thing as @Shamool  when using 2.11.6. haven't tried 2.11.7 tho
using 2.11.3 for now to avoid the issue

**Comment by Shamool** [comment]
Eh, V2.11.7 also doesn't work for me.

@Masuzu Any help you can render here?

**Comment by aysien** [comment]
![error](https://user-images.githubusercontent.com/31809167/37922214-f251a918-3166-11e8-8c45-77ba1fa2816f.jpg)

My bot always stuck in this location.
Its happen since 2.11.6 and still happen in 2.11.7
But its totally fine in 2.11.5.
It might related to Improved summon retreat feature on 2.11.6.
Im using Chromium Version 63.0.3225.0 (Developer Build) (64-bit).
Please take a look about this case.

**Comment by Shamool** [comment]
@Einhalkzt Could you zip up your V2.11.3 and pass it over to me for now?

**Comment by aysien** [comment]
@Shamool 
You can download the previous version from here https://github.com/Masuzu/ZooeyBot/tags

**Comment by Shamool** [comment]
@aysien Ehhh..... Really? I extracted it and all I got was this

![zooey](https://user-images.githubusercontent.com/30111700/37942285-f6b12fb8-31a4-11e8-938e-9cc2018a6118.PNG)

**Comment by aysien** [comment]
@Shamool 
Ah, My bad, normally on Git we can download the previous version of the released, but it seems Masuzu closed the access to download the files. 
I was able to download the previous version from there.
I achieved almost all version, but I'm at the office right now so I cant access my PC at home.
Sorry.

@Masuzu 
I suggest opening the previous version download (at least up to 5 latest version) because the new version update does not always work properly and it will bad for users if they cannot run the bot.

**Comment by Einhalkzt** [comment]
@shamool https://goo.gl/ppDHe9

**Comment by Shamool** [comment]
Much thanks Einhalkzt

**Comment by Masuzu** [comment]
Hello,
I will address the issue causing Zooey to be stuck on this screen:
https://user-images.githubusercontent.com/31809167/37922214-f251a918-3166-11e8-8c45-77ba1fa2816f.jpg

**Comment by Masuzu** [comment]
@aysien About leaving past versions, I will consider reintroducing it, but I have no guarantees, due to external reasons which I can't discuss here unfortunately.

**Comment by Shamool** [comment]
Thanks for the response and help Masuzu, appreciate it.
